app/src/main/java/com/github/multimatum_team/multimatum/activity/MainActivity.kt

```diff
@@ -74,6 +73,8 @@ class MainActivity : AppCompatActivity() {
         setCurrentTheme()
     }
 
+    val context = this
```

Comment:
What's the purpose of defining this variable?

Reply:
You can simply do `this@MainActivity` instead of defining an intermediary variable

Reply:
ok